benefits/eligibility/forms.py

```diff
@@ -108,6 +108,7 @@ def __init__(
         self.title = title
         self.headline = headline
         self.blurb = blurb
+        self.classes = "eligibility-verification-form"
 
```

Comment:
I did not add a class for the Enrollment Flow Selection form, because it doesn't need one. This form actually did _not_ need the gap declaration at all!